### DISCUSSION TITLE
Is it possible to use postgre from another source in google cloud run? What needs to be changed in configuration? like isUnixSocket this one required?

### DISCUSSION THREAD

**Discussion by ps6067966** [serverpod/serverpod#999] [Q&A] [answered]
![image](https://github.com/serverpod/serverpod/assets/26248740/3f83b087-bcfe-45a5-9ace-2757fdff517c)
Is it possible to use postgre from another source in google cloud run? What needs to be changed in configuration? like isUnixSocket this one required?

**Comment by vlidholt** [comment, marked as answer]
The unix socket is required if you are running on Cloud Run and connecting to a Postgres database that does not have a publicly available port. If your database is accessible from the internet you can just use the domain/ip-address instead.
